### ISSUE TITLE
sourceNumber is not in the envelope anymore

### ISSUE THREAD

**Issue by stack74** [bbernhard/signal-cli-rest-api#273]
### The problem

When I receive a message the data looks like this.
I am missing the sourceNumber / phonenumber of the sending device

```json
[{
  envelope: {
    source: '671aed99-4e23-4794-bcbb-0b2e1eae6413',
    sourceNumber: null,
    sourceUuid: '671aed99-4e23-4794-bcbb-0b2e1eae6413',
    sourceName: 'My Nickname',
    sourceDevice: 1,
    timestamp: 1661513770123,
    dataMessage: '[Object]'
  },
  account: '+123456789',
  subscription: 0
}]
```

### Are you using the latest released version?

- [X] Yes

### Have you read the troubleshooting page?

- [X] Yes

### What type of installation are you running?

signal-cli-rest-api Docker Container

### In which mode are you using the docker container?

JSON-RPC Mode

### What's the architecture of your host system?

x86-64

**Comment by stack74** [comment]
Posted on signal-cli:
https://github.com/AsamK/signal-cli/issues/1002

**Comment by git-developer** [comment]
I can confirm that the problem
1. exists using docker image `bbernhard/signal-cli-rest-api:0.62` and
2. may be fixed by using in current `master` of signal-cli (Commit `c7aa983`).

Steps for a workaround (until a fix is released):
1. Download latest signal-cli build artifact from [GitHub actions](https://github.com/AsamK/signal-cli/actions) (at the time of writing: [here](https://github.com/AsamK/signal-cli/actions/runs/2950498165), at the bottom of the page, named `signal-cli-archive-18`)
2. Unzip directory `signal-cli-0.10.11`
3. Start docker with a volume mount that overrides the signal-cli cirectory, e.g. in `docker-compose.yml`:

```yaml
    volumes:
      - "./signal-cli-config:/home/.local/share/signal-cli"
      - "./signal-cli-0.10.11:/opt/signal-cli-0.10.9:ro"
```

**Comment by Mincka** [comment]
It looks like the latest published version of `signal-cli` fixes this issue 🎉
https://github.com/AsamK/signal-cli/releases/tag/v0.11.0

> Fixed
> Fix an issue where messages were sent without sender phone number

Now let's wait for the next release of `signal-cli-rest-api` 😀

**Comment by Mincka** [comment]
I can confirm that this issue is fixed in 0.64. Thank you!
